core/src/main/java/com/linecorp/armeria/common/util/CompositeException.java

```diff
@@ -314,13 +322,20 @@ static final class ExceptionOverview extends RuntimeException {
 
         private static final long serialVersionUID = 3875212506787802066L;
 
+        private static final ReentrantLock reentrantLock = new ReentrantLock();
+
         ExceptionOverview(String message) {
             super(message);
         }
 
         @Override
-        public synchronized Throwable fillInStackTrace() {
-            return this;
+        public Throwable fillInStackTrace() {
+            reentrantLock.lock();
```

Comment:
I think we don't need a lock for this.

Reply:
delete lock
[3727da5](https://github.com/line/armeria/pull/4551/commits/3727da5ec5b3636d22f4979cf5617fca8c615df9)